lib/chewy/type/crutch.rb

```diff
@@ -23,7 +24,11 @@ def #{name}
 
       module ClassMethods
         def crutch name, &block
-          self._crutches = _crutches.merge(name.to_sym => block)
+          if block
+            self._crutches = _crutches.merge(name.to_sym => block)
+          else
+            self._crutches_classes = _crutches_classes + [name]
```

Comment:
This is a class_attribute, we have to assign different object every time otherwise, it will be rewritten by ancestors